src/modules/cluster/cluster-list/cluster-list.scss

```diff
@@ -12,14 +12,6 @@
         text-transform: capitalize;
     }
 
-    .counts {
-        color: $color-boston-blue;
-
-        &:hover {
-            cursor: pointer;
-        }
-    }
-
```

Comment:
@gnehapk Also anchor tag does this already, so we anyway don't need this extra class.